scripts/copy-build-to-dist.mjs

```diff
@@ -141,9 +141,9 @@ async function getPackageBuildPaths(moduleRootDir) {
       if (path.basename(moduleDir) === "@remix-run") {
         packageBuilds.push(...(await getPackageBuildPaths(moduleDir)));
       } else if (
-        /node_modules\/@remix-run\//.test(moduleDir) ||
-        /node_modules\/create-remix/.test(moduleDir) ||
-        /node_modules\/remix/.test(moduleDir)
+        /node_modules[/\\]@remix-run[/\\]/.test(moduleDir) ||
+        /node_modules[/\\]create-remix/.test(moduleDir) ||
+        /node_modules[/\\]remix/.test(moduleDir)
```

Comment:
This fixes an issue where the built packages weren't being copied to the `dist` directories on Windows. This tripped me up because it meant that the `readConfig` tests passed on Mac/Linux but failed on Windows. This is because `readConfig` detects the presence of the `@remix-run/css-bundle` package, but this package wasn't being detected on Windows.